x-pack/plugins/maps/public/_main.scss

```diff
@@ -1,10 +1,11 @@
 @import '@elastic/eui/src/global_styling/variables/header';
+$headerOffset: $euiHeaderHeightCompensation * 2;
```

Comment:
It would be great to store this more globally so if it were to ever change again, all the apps would auto-update and they wouldn't have to individually import the `$euiHeaderHeightCompensation` variable.